ViAn/GUI/analysisslider.h

```diff
@@ -4,25 +4,37 @@
 #include <QWidget>
 #include <QSlider>
 #include <vector>
+#include "Project/Analysis/analysis.h"
 
 
 class AnalysisSlider : public QSlider {
+    Q_OBJECT
     bool m_blocked = false;
     bool m_was_paused = false;
+    bool m_show_pois = false;
 public:
     explicit AnalysisSlider(Qt::Orientation orientation, QWidget *parent = 0);
-    void add_slider_interval(int start_frame, int end_frame);
-    void clear_slider();
+
+    int last_poi_end;
```

Comment:
will this variable always get a value before being used? otherwise give it some init value here